src/physics/src/elastic_cable.C

```diff
@@ -91,33 +91,36 @@ namespace GRINS
 
             if( name == std::string("stress") )
               {
-                // sigma_xx, sigma_xy, sigma_yy, sigma_yx = sigma_xy
-                // sigma_zz = 0 by assumption of this Physics
-                _stress_indices.resize(3);
+                // sigma_xx
+                // sigma_yy=sigma_zz = 0 by assumption of this Physics
+                _stress_indices.resize(1);
 
                 this->_stress_indices[0] = postprocessing.register_quantity("stress_xx");
```

Comment:
We need to differentiate the name between the membrane version is well. `cable_stress`?